src/matrix/inversewishart.jl

```diff
@@ -110,7 +110,8 @@ end
 function invwishart_logc0(df::Real, Ψ::AbstractPDMat)
     h_df = df / 2
     p = size(Ψ, 1)
-    -h_df * (p * typeof(df)(logtwo) - logdet(Ψ)) - logmvgamma(p, h_df)
+    logdet_Ψ = logdet(Ψ)
```

Comment:
I think the `Float32` issue could be fixed e.g. in the following way:
```suggestion
    logdet_Ψ, _df = promote(logdet(Ψ), df)
    h_df = _df / 2
```